test- minor team bio change

annapeterson816 opened the pull request:
testing this new way to update the git and the different branches!

*All I changed was one spelling/grammatical error on the team page in Paiton's section.

annapeterson816 committed:
test- minor team bio change

testing this new way to update the git and the different branches!

lurein reviewed (changes requested):
Hey Anna, the code looks good but the one issue is that you committed directly into staging, that's dangerous because the assumption in our workflow is that all of the code in staging has been approved and can be merged into master whenever - in this case this would be harmless but this can cause some dangerous side effects unchecked. Could you make a branch off of staging? The instructions for this are in the steps in the wiki. If you branch off of staging then push these same changes to that branch, then make a different PR to merge into staging from your new branch. In essence nothing should ever be committed straight to staging without review.

annapeterson816 committed:
removed last name

lurein committed:
Merge pull request #3 from lurein/donationplatform

test 2- removed last names from donation platform

annapeterson816 committed:
Added Will to team page

Added Will Peters to the team page. He was unsure if his title is actually "Project Leader", so wanted you to double check it and change if needed!

lurein committed:
Merge pull request #4 from lurein/misc

Added Will to team page
